libtransmission/rpc-server.cc

```diff
@@ -827,7 +825,9 @@ static void stopServer(tr_rpc_server* server)
         unlink(address + std::size(TrUnixSocketPrefix));
     }
 
-    tr_logAddInfo(fmt::format(_("Stopped listening on {address}"), fmt::arg("address", tr_rpc_address_with_port(server))));
+    tr_logAddInfo(fmt::format(
+        _("Listening for RPC and Web requests on '{address}'"),
```

Comment:
Somehow this message changed from negative to positive.

Reply:
ecc7c8c4b472e0a4b28982bb5b06219648b8df66